GearSpawner/LootTableManager.cs

```diff
@@ -20,66 +18,52 @@ internal static void AddLootTableEntry(string lootTable, LootTableEntry entry)
 		lootTableEntries[normalizedLootTableName].Add(entry.Normalize());
 	}
 
-	private static void AddEntries(LootTable lootTable, List<LootTableEntry> entries)
+	private static string GetNormalizedLootTableName(string lootTable)
```

Comment:
Move this method to the bottom so that the diff looks decent and I can see what's going on.